src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -116,9 +116,10 @@ public function upload(): Response
         return $this->request($this->request);
     }
 
-    public function buildCreateRequest(): void
+    public function buildCreateRequest(?string $status = null): void
     {
-        $this->request = Request::create($this->resource, $this->sharedStorage->get('token'));
+        $token = $status === null ? $this->sharedStorage->get('token') : $status;
+        $this->request = Request::create($this->resource, $token);
```

Comment:
Maybe we should get token only if it exists

Reply:
We need to add some token, In this case we need wrong token, I will try to improve it.